css/selectors/where.json

```
@@ -7,7 +7,14 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:where",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "83",
```

This feature seems to be somewhat older than your test shows. It looks like [this change](https://storage.googleapis.com/chromium-find-releases-static/e4a.html#e4a8dda8926d36a02e3f3bc3d20d839bb71ede5c) renamed it from `is()` to `:where()` in Chrome 72.

```suggestion
              "version_added": "72",
```

---

You found the change, nice! Tried to but didnt find it...